.github/workflows/build-and-publish-docker-image.yml

```diff
@@ -0,0 +1,34 @@
+name: Docker Image CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'release/10.0.x'
```

Comment:
Why this branch specifically? Do we intend to keep it around? Or is this just for testing purposes?

Reply:
This is a fork of the redash release branch that the docker image we currently use is built from. Therefore, all changes we make will be in this branch. Any pull requests should be made against this branch.